src/main/java/com/addthis/meshy/MeshyServer.java

```diff
@@ -77,6 +78,10 @@
     private static final boolean allowPeerLocal = Parameter.boolValue("meshy.peer.local", true);
     private static final int autoMeshTimeout = Parameter.intValue("meshy.autoMeshTimeout", 60000);
 
+    private static final int retryPeersUnitDelay = Parameter.intValue("mss.mesh.retry.peers.unitDelay", 900);
+    private static final int retryPeersMaxDelayUnit = Parameter.intValue("mss.mesh.retry.peers.maxDelayUnit", 3);
+    private static final int retryPeersMaxRetries = Parameter.intValue("mss.mesh.retry.peers.maxRetries", 5);
```

Comment:
How about `mss.mesh.connection.retry.maxDelayUnit`, `mss.mesh.connection.retry.unitDelay`, `mss.mesh.connection.retry.maxRetries`?

Reply:
Feel free to disagree, not sure if mine or yours is better.